rcl/Doxyfile

```diff
@@ -0,0 +1,17 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl"
+PROJECT_NUMBER         = beta1
```

Comment:
I would say "master". And when doing the documentation building run temporary replace the string locally before invoking Doxygen. But no need to check the specific build number in.